docs/content/doc/usage/email-setup.en-us.md

```diff
@@ -37,6 +37,9 @@ MAILER_TYPE   = sendmail
 SENDMAIL_PATH = /usr/sbin/sendmail
 ```
 
+Then configure `sendmail` (or mailer that provide the command) to send outgoing email without accepting incoming email from outside. For example configuration,
+see [Postfix with SMTP Relay](#sendmail-example-postfix-with-smtp-relay).
```

Comment:
The configuration of a postfix relay requires a lot of documentation and is dependent on the default provided by the package after installing it on a given Operating System. The example provided below is unlikely to work in most cases. It would however be useful to link to the postfix documentation for more information (for instance http://www.postfix.org/SMTPD_ACCESS_README.html).